eslint.config.js

```diff
@@ -15,7 +33,15 @@ export default tseslint.config(
   },
   {
     rules: {
+      'import-x/no-unresolved': 'off', // TODO: figure out how to resolve @/ imports
+      'import-x/no-named-as-default-member': 'off',
+      'import-x/no-named-as-default': 'error',
+
       'no-console': ['error', { allow: ['info', 'warn', 'error'] }],
+      'no-throw-literal': 'error',
+      'no-useless-concat': 'error',
+      'prefer-template': 'error',
+      eqeqeq: 'error',
```

Comment:
What does this do :dagger:

Reply:
eqeqeq forces triple equal check everywhere because double equals sucks